arviz/plots/forestplot.py

```diff
@@ -372,10 +371,7 @@ def make_bands(self, ax):
                 prev_color_index = self.colors.index(color)
                 y_prev = y
 
-        if plotter is None:
-            offset = 1
```

Comment:
Given that if this is True, the `y_prev` would be `None` which would raise an Exception.